### PR TITLE
Adding changelog entries

### PR DESCRIPTION
**Pull request by LikeTheSalad**

### CODE REVIEW

File: .github/scripts/draft-change-log-entries.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+else
+  range="v$major.$((minor - 1)).0..HEAD"
+fi
+
+echo "## Unreleased"
+echo
+echo "### Migration notes"
+echo
+echo
+echo "### 🌟 New javaagent instrumentation"
```
Comment (LikeTheSalad) [inline .github/scripts/draft-change-log-entries.sh:31]: Hmm, I guess we won't need this here, I'll update it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
 # OpenTelemetry Android Changelog
 
+## Unreleased
+
```
Comment (breedx-splk) [inline CHANGELOG.md:4]:
```suggestion
## Unreleased

⚠️⚠️⚠️ There are considerable breaking changes in this release.

Breaking changes include considerable restructuring of the overall project layout. This provides a much more modularized project that publishes more granular instrumentation modules. Note that as a result of this, the topmost dependency is changing its name to `io.opentelemetry.android:android-agent`.
```

**Review by breedx-splk — APPROVED** [review]
Onward and upward!

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+⚠️⚠️⚠️ There are considerable breaking changes in this release.
+
+Breaking changes include considerable restructuring of the overall project layout. This provides a much more modularized project that publishes more granular instrumentation modules. Note that as a result of this, the topmost dependency is changing its name to `io.opentelemetry.android:android-agent`.
+### 📈 Enhancements
```
Comment (breedx-splk) [inline CHANGELOG.md:8]:
```suggestion

### 📈 Enhancements
```

**Pull request merged**